storage/bucketmgr/main_test.go

```
@@ -0,0 +1,70 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
```

Fails fast on CI now with #113 at least